File: common_library/src/main/java/com/phone/common_library/manager/ThemeManager.java
package com.phone.common_library.manager;

import android.content.Context;
import android.content.res.TypedArray;

import com.phone.common_library.BaseApplication;

public class ThemeManager {

    public static int getThemeColor(int attrRes) {
        TypedArray typedArray = BaseApplication.getInstance().obtainStyledAttributes(new int[]{attrRes});
        int color = typedArray.getColor(0, 0xffffff);
        typedArray.recycle();
        return color;
    }
}

File: library_common/src/main/java/com/phone/common_library/manager/ThemeManager.kt
package com.phone.common_library.manager

import com.phone.common_library.BaseApplication

class ThemeManager {

    fun getThemeColor(attrRes: Int): Int {
        val typedArray = BaseApplication.get().obtainStyledAttributes(intArrayOf(attrRes))
        val color = typedArray.getColor(0, 0xffffff)
        typedArray.recycle()
        return color
    }
}